cfme/login.py

```diff
@@ -14,14 +14,17 @@
 
 from cfme.base.credential import Credential
 from fixtures.pytest_store import store
-from utils import conf
+from utils import conf, version
 from utils.log import logger
 from widgetastic_patternfly import Button, Input, FlashMessages
 from .base.login import BaseLoggedInPage
 
 
 class LoginPage(View):
-    flash = FlashMessages('div#flash_text_div')
+    if version.current_version <= 5.7:
+        flash = FlashMessages('div#flash_text_div')
+    else:
+        flash = FlashMessages('div.flash_text_div')
```

Comment:
Instead of that use this construct:
```python
flash = VersionPick({
    Version.LOWEST: FlashMessages('div#flash_text_div'),
    '5.8': FlashMessages('div.flash_text_div')})
```

Or you can use it like this:
```python
flash = FlashMessages(VersionPick({Version.LOWEST: 'div#flash_text_div', '5.8': 'div.flash_text_div'}))
```